jackson/src/main/java/io/gravitee/definition/jackson/datatype/api/deser/StepDeserializer.java

```diff
@@ -41,6 +41,10 @@ public Step deserialize(JsonParser jp, DeserializationContext ctxt) throws IOExc
         step.setName(node.path("name").asText());
         step.setDescription(node.path("description").asText());
         step.setPolicy(node.get("policy").asText());
+        final JsonNode executionCondition = node.get("executionCondition");
```

Comment:
Just `condition` should be sufficient, isn't it ?

Reply:
done :)